### ISSUE TITLE
Bug when importing PDF to Obsidian.

### ISSUE THREAD

**Issue by DudeThatsErin** [omnivore-app/omnivore#2317]
I would expect to see a copy of the PDF saved to Obsidian and not the information from the PDF copied to Obsidian. It looks ridiculous lol

![image](https://github.com/omnivore-app/omnivore/assets/2681022/9aeb5286-cac1-439b-945e-7c8f5dc24ac5)

This screenshot shows exactly what I am seeing on my iPad.

**Comment by jacksonh** [comment]
Thanks, it looks like it grabbed our search content, not the actual PDF document here. Does this happen with all PDFs or happened with just this one?

**Comment by sywhb** [comment]
Hey @DudeThatsErin , there is another variable in the template: `fileAttachment` which could be used to export PDF as an attachment in Obsidian. 

I can create an attachment using below template.

`![[{{{fileAttachment}}}]]`

**Comment by sywhb** [comment]
@jacksonh I guess I should set `content` to `null` if the `type` of the item is `FILE` so we won't pull the search content from the PDF

**Comment by jacksonh** [comment]
@sywhb there is a contentReader attribute that should be set to WEB for any web article. That is probably the best way to do it.

**Comment by sywhb** [comment]
@jacksonh sorry, I mean in Obsidian template

**Comment by jacksonh** [comment]
Yeah i think when generating the template values you can do something like:

```
content" article.contentReader === 'WEB' ? article.content : null
```

**Comment by sywhb** [comment]
Yeah!
